packages/rainbowkit/package.json

```diff
@@ -60,6 +60,7 @@
     "vitest": "^0.5.0"
   },
   "dependencies": {
+    "@infinitywallet/infinity-connector": "^1.0.5",
```

Comment:
Our current approach here is to rely solely on `@wagmi/connectors` for peer dependency connectors. Please first work with the `wagmi` team for upstream support, and then we'll revisit. https://github.com/wagmi-dev/references/tree/main/packages/connectors